Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.11.6-alpine3.18
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
Consider copying only the `requirements.txt` file first and running the `pip install` command before copying the rest of the application files. This will help leverage Docker's caching mechanism and avoid reinstalling dependencies if only application code changes.